addons/io_scene_gltf2/__init__.py

```diff
@@ -336,6 +336,18 @@ def invoke(self, context, event):
                 self.report({"ERROR"}, "Loading export settings failed. Removed corrupted settings")
                 del context.scene[self.scene_key]
 
+        import sys
+        self.user_extensions = []
+        for addon_name in bpy.context.preferences.addons.keys():
+            try:
+                extension_ctor = sys.modules[addon_name].glTF2UserExtension
```

Comment:
Maybe put `Export` in the name, like `glTF2ExportUserExtension`? If that's kinda wordy, maybe `glTF2ExportExtension`?

Just for consistency with a hypothetical `glTF2ImportExtension` at some point in the future (julienduroure mentioned that somewhere).

Reply:
Done